modules/end-user-guide/pages/mounting-volumes.adoc

```diff
@@ -0,0 +1,66 @@
+:navtitle: Mounting Volumes
+:keywords: user-guide, configuring, user, volumes, persistent, volume, claim, mounting, mount
+:page-aliases:
+
+[id="mounting-volumes_{context}"]
+= Mounting Volumes
+
+You can mount a Persistent Volume Claim (PVC) to your {devworkspace} containers. A PVC is used to request persistent storage resources which are abstracted from the user. This section describes how to automatically mount an existing PVC to all {devworkspace} containers.
+
+.Prerequisites
+
+* A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+* For CLI users: The `{orch-cli}` command line tool is installed in the operating system you are using.
+
+.Procedure
+
+. Determine an existing PVC in your user {orch-namespace} to mount to all {devworkspace} containers.
+
+. Set the `controller.devfile.io/mount-to-devworkspace` label to `'true'`.
+
++
+CLI users can apply the label by running:
+
++
+[subs="+quotes,+attributes,+macros"]
+----
+$ {orch-cli} -n __<user_{orch-namespace}>__ label persistentvolumeclaim __<PVC_name>__ controller.devfile.io/mount-to-devworkspace=true
+----
+
+. Optionally configure how the PVC is mounted using the following annotations:
++
+.Optional annotations
+,===
+ Annotation, Description
+ `controller.devfile.io/mount-path`, Path to where the PVC should be mounted. Defaults to `/tmp/<pvc-name>`.
+ `controller.devfile.io/read-only`,  Set to `'true'` or `'false'` to specify whether the PVC should be mounted as read-only. Defaults to `'false'` resulting in the PVC being mounted as read-write.
+,===
+
+See the following example:
+
+.Mounting a read-only PVC
+====
+[source,yaml,subs="+quotes"]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-name
+  labels:
+    controller.devfile.io/mount-to-devworkspace: 'true'
+  annotations:
+    controller.devfile.io/mount-path: /tmp/my-directory
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 3Gi
+  volumeName: pv-name
+  storageClassName: manual
+  volumeMode: Filesystem
+----
+
+When a {devworkspace} starts, the PVC will be mounted at `/tmp/my-directory` in the {devworkspace} containers.
```

Comment:
Just double-checking: the directory is exactly `/my-directory`, correct? (And this isn't a user-replaced value.)